client/src/components/HolidayShow.js

```diff
@@ -10,8 +11,18 @@ const HolidayShow = (props) => {
     reviews: [],
   })
 
+  const [currentUser, setCurrentUser] = useState(undefined)
+  const fetchCurrentUser = async () => {
+    try {
+      const user = await getCurrentUser()
+      setCurrentUser(user)
+    } catch (err) {
+      setCurrentUser(null)
+    }
+  }
```

Comment:
Since `App.js` is already fetching for the current user, we could instead set up the route for this component to receive the `currentUser` as props to avoid an additional fetch request!

```js
// App.js

...

        <Route exact path="/holidays/:id">
          <HolidayShow user={currentUser} />
        </Route>
```